src/Couchbase.Lite.Shared/API/Document/Document.cs

```diff
@@ -122,6 +122,8 @@ internal C4DocumentWrapper c4Doc
         /// </summary>
         public ulong Sequence => ThreadSafety.DoLocked(() => c4Doc?.HasValue == true ? c4Doc.RawDoc->selectedRev.sequence : 0UL);
 
+        public DateTimeOffset? Expiration => null;
```

Comment:
This will always return `null`

Reply:
I wonder if we need to keep this in the document level? We do have method GetDocumentExpiration in database level.